lib/hex_web/email/ses.ex

```diff
@@ -3,8 +3,21 @@ defmodule HexWeb.Email.SES do
   require Logger
 
   def send(to, subject, body) do
-    HexWeb.Throttle.wait(HexWeb.SESThrottle)
-    do_send(to, subject, body)
+    ses_rate = Application.get_env(:hex_web, :ses_rate) |> String.to_integer
+
+    recipient_chunks(to, ses_rate)
+    |> Enum.each(fn recipients ->
+      HexWeb.Throttle.wait(HexWeb.SESThrottle, length(recipients))
+      do_send(recipients, subject, body)
+    end)
+  end
+
+  defp recipient_chunks(recipients, limit) do
+    if length(recipients) <= limit do
+      [recipients]
+    else
+      Enum.chunk(recipients, limit)
```

Comment:
I was bitten by this myself today. Calling it this way will drop elements from the list, the correct way to call it is: `Enum.chunk(recipients, limit, limit, [])`.